test/unit/processing/components/get-schedule-dates.test.js

```diff
@@ -93,7 +93,12 @@ describe('get schedule dates', () => {
 
   test('should insert top up adjustment after last outstanding payment in original schedule', () => {
     const schedule = getScheduleDates(previousPaymentSchedule, newPaymentScheduleTopUp, deltaValue)
-    expect(schedule[2].period).toEqual('Adjustment')
+    expect(schedule[2].period).toEqual(moment().format('MMM YYYY'))
+  })
+
+  test('Adjustment payment Type must be Immediate payment', () => {
```

Comment:
The objects used in this file are not defined in the mock-objects folder.

Meaning they are susceptible to drift becuz they do not use defined mock values or constants as appropriate, as evidenced here -- the period for these mock objects are given in the old(?) format 'September to December 2020' rather than the new(?) format 'Sep-Dec 2020'

Ideally I feel these objects should be defined in the mock-objects as was started with `mock-payment-schedule`. However, I feel for this PR we can ignore doing so in the interest of time. Although, I would ask we update the periods.

Also in: `test/unit/processing/payment-schedule/get-payment-schedule.test.js`

Reply:
Missed point on changing the periods in this file and `test/unit/processing/payment-schedule/get-payment-schedule.test.js` to match the new format

Reply:
Added as debt ticket